src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpec.scala

```diff
@@ -356,6 +356,25 @@ class OldUserServiceSpec
     }
   }
 
+  it should "record the date that the user registered if they were previously invited" in {
+    assume(databaseEnabled, databaseEnabledClue)
+    // Arrange
+    val user = genWorkbenchUserGoogle.sample.get
+    assume(databaseEnabled, databaseEnabledClue)
+
+    // Act
+    val createdAt = Instant.now()
+    service.inviteUser(user.email, samRequestContext).unsafeRunSync()
+
+    val userStatus = service.createUser(user, samRequestContext).unsafeRunSync()
+
+    // Assert
+    val maybeUser = dirDAO.loadUser(userStatus.userInfo.userSubjectId, samRequestContext).unsafeRunSync()
```

Comment:
I dont really like that this needs to talk to the directoryDao, but I think this is more an issue with the fact that createUser returns a `UserStatus` and not a `SamUser`. So I think this is fine for now, but I think going forward, we should change `createUser` to return a `SamUser` and refactor so that the endpoints that are asking for a UserStatus can still get it